test/unit/lib/utils.js

```diff
@@ -0,0 +1,29 @@
+const chai = require('chai');
+const utils = require('../../../app/lib/utils');
+
+const expect = chai.expect;
+
+describe('utils', () => {
+  describe('deepClone', () => {
```

Comment:
I didn't take it as derogatory, merely wondering if there was some additional context I didn't understand. I was wondering if there was a better (rich man's) version...

I was going to use `Object.assign({}, obj)` but didn't because it is a shallow clone. And yes, `JSON.parse(JSON.stringify(obj))` isn't very clever and doesn't include functions but it only needs to deal with the data so I figured it would be fine. Hence some very basic tests to check it does the data clone and the implementation could be swapped out if/when required.